cargo-dist/templates/ci/github/partials/publish_github.yml.j2

```diff
@@ -0,0 +1,37 @@
+      # Create a GitHub Release while uploading all files to it
+      - name: "Download GitHub Artifacts"
+        uses: actions/download-artifact@v4
+        with:
+          pattern: artifacts-*
+          path: artifacts
+          merge-multiple: true
```

Comment:
I was gonna complain this is redundant (we end up downloading all the artifacts twice) but now I see we've always been doing this and there's some reason for why, so nvm.

Reply:
I feel like this kinda branch always exposes those.